### PR TITLE
separated deployment script, using a basic bash script to call the ap…

### PR DESCRIPTION
**Pull request by shaunclarke2333**
…p config script

### CODE REVIEW

File: level2/variables.tf [hunk 1]
```diff
@@ -12,5 +12,5 @@ variable "key_name" {
 
 variable "keys_and_salts" {
   description = "Authentication unique keys and salts"
-  sensitive = true
+  sensitive   = true
 }
```
Comment (vladimirmukhin) [inline level2/variables.tf]: Please add trailing newline

File: level3/datasources.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#This remote data source is for main vpc and main-public0 id
+data "terraform_remote_state" "level1-main-vpc" {
+  backend = "s3"
+
+  config = {
+    bucket = "main-backend-state-bucket"
+    key    = "level1/terraform.tfstate"
+    region = "us-east-1"
+  }
+}
```
Comment (vladimirmukhin) [inline level3/datasources.tf]: Please add trailing newline

**Review by vladimirmukhin — APPROVED** [review]
Please fix trailing newlines. Otherwise all good

File: level3/datasources.tf (hunk 2 above)
Comment (shaunclarke2333) [reply level3/datasources.tf]: added trailing new lines

File: level2/variables.tf (hunk 1 above)
Comment (shaunclarke2333) [reply level2/variables.tf]: added trailing new lines

**Pull request merged**